core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -796,7 +796,7 @@ enum class UniqueType(
     OneTimeRevealCrudeMap("From a randomly chosen tile [positiveAmount] tiles away from the ruins, reveal tiles up to [positiveAmount] tiles away with [positiveAmount]% chance", UniqueTarget.Ruins),
     OneTimeGlobalAlert("Triggers the following global alert: [comment]", UniqueTarget.Triggerable), // used in Policy
     OneTimeGlobalSpiesWhenEnteringEra("Every major Civilization gains a spy once a civilization enters this era", UniqueTarget.Era),
-    OneTimeSpiesLevelUp("Promotes all spies", UniqueTarget.Triggerable),  // used in Policies, Buildings
+    OneTimeSpiesLevelUp("Promotes all spies [amount] times", UniqueTarget.Triggerable),  // used in Policies, Buildings
```

Comment:
```suggestion
    OneTimeSpiesLevelUp("Promotes all spies [amount] time(s)", UniqueTarget.Triggerable),  // used in Policies, Buildings
```

What do you think?

Reply:
I looked at that before and decided to keep it consistent with the other uniques:

- Cities are razed [amount] times as fast
- When spreading religion to a city, gain [amount] times the amount of followers of other religions as [stat]
- Upon capturing a city, receive [amount] times its [stat] production as [civWideStat] immediately
- [amount] times

Compared to only

- [amount] additional time(s)


Either way works, if you want to make that transition.

Reply:
There's a difference between "times" in the meaning of "A * B" - which is always "times", you don't say "1 time 5" you say "1 times 5" - and in the meaning of "repeating a certain amount", where you would say "1 time"

In all but the last one, it's the mathematical meaning
In the last one, it *is* the repeating meaning, but "1 times" doesn't make sense *for that modifier*, since it would have zero impact